docs/semgrep-appsec-platform/azure-pr-comments.md

```diff
@@ -59,6 +59,52 @@ PR comments are enabled by default for users who have connected their Azure DevO
 Only rules set to the **Comment** and **Block** rule modes in the [Policies page](https://semgrep.dev/orgs/-/policies) create PR comments.
 :::
 
+In the Azure Pipelines configuration file, you must export the `SEMGREP_REPO_URL` variable to enable PR comments. Ensure that the namespace as part of the variable's value follows the format `{organization}/{project}`:
+
+```
+# example
+export SEMGREP_REPO_URL="https://dev.azure.com/{organization}/{project}/_git/{project}"
+```
+
+<details>
+<summary>Click to see a sample workflow file</summary>
+
+```yaml
+pool:
+  vmImage: ubuntu-latest
+variables:
+  - group: Semgrep_Variables
+steps:
+  - checkout: self
+    clean: true
+    fetchDepth: 100000
+    persistCredentials: true
+  - script: >
+      python -m pip install --upgrade pip
+
+      pip install semgrep
+
+      if [ $(Build.SourceBranchName) = "main" ]; then
+          echo "Semgrep full scan"
+          semgrep ci
+      elif [ $(System.PullRequest.PullRequestId) -ge 0 ]; then
+          echo "Semgrep diff scan"
+          export SEMGREP_PR_ID=$(System.PullRequest.PullRequestId)
+          export SEMGREP_REPO_URL="https://dev.azure.com/{organization}/{project}/_git/{project}"
```

Comment:
We also recommend that they set the `SEMGREP_REPO_NAME` environment variable. It can be set via 
```
export SEMGREP_REPO_NAME="{organization}/${SYSTEM_TEAMPROJECT}/${BUILD_REPOSITORY_NAME}"
```
Similarly the `SEMGREP_REPO_URL` can be simplified to 
```
export SEMGREP_REPO_URL="https://dev.azure.com/{organization}/${SYSTEM_TEAMPROJECT}/_git/${BUILD_REPOSITORY_NAME}"
```